Meetups/19/2024-10-14-minutes.md

```diff
@@ -0,0 +1,215 @@
+# Meeting Minutes
+
+:date: **Date:** 23 May 2024
+
+## Information
+
+:computer: <https://docs.google.com/presentation/d/1E8CsW-45U9YBcEFFEUQ5u-K6n4j4j8nkqNRScQRjuUI/edit?usp=sharing>
+
+### :bust_in_silhouette: Participants
+
+Total: 29 Participants
+
+- Ege Korkan
+- Cristiano Aguzzi
+- Thomas Jaeckle
+- Sebastian Kaebisch
+- Roman Binkert
+- Christian Winkler
+- Christian Glomb
+- Matthias Langner
+- Michael McCool
+- Daniel Peintner
+- Alexander Schmidt
+- York Schmidtke
+- Sergio Sevilieanu
+- Hossein Rimaz
+- John Caipa
+- Kaz Ashimura
+- Mikinobu
+- Lorenzo Gigli
+- Tanil Ergin
+- Flavia De Andrade Pereira
+- Sibla Wilfried
+- Thomas Schwarz
+- Christian Paul
+- Marques Bruno
+- Minyong Li
+- Michael Koster
+- Kunihiko Toumura
+- Alexander Power
+- Cai Li
+- Matthias Langner
+
+:writing_hand: **Scribe:** Cris
+
+----
+
+## Minutes
+
+### Introduction
+
+Ege: Welcome to the 19th meetup!
+
+### Agenda
+
+Ege: as always some news and later we will start the talk
+
+### News
+
+Ege: we will host three new Meetups! and we have the WoT week during the 25-29. If you want to have a talk please contact us and we can arrange a new event.
+
+### WoT week in Munich
+
+Ege: it will have a Plug Fest event to play with different devices and test interoperability between them. Full week event but the last two days will be only for standardization activities
+
+### Using thing Models
+
+Ege: We have Thomas Jaeckle who is currently working at beyonnex.io previously in Bosh where he also contributed to the WoT standard specifically in the WoT Model space.
+
+Thomas: thanks for the introduction, I will talk about how we use Thing Model in the housing sector.
+
+Thomas: we are from beyonnex.io a new IoT startup (younger than 3 years) we are digitalizing the real estate sector. More than 100 employees spread across Europe.
+
+Thomas: We have a B2B model and our customers are housing companies however residents use our devices as a services. No need to think about battery connectivity or maintenance.
+
+Thomas: the group started from the submetering devices to collect and allocate head/water consumptions among residents. We also offer backend and application services.
+
+Thomas: the devices are connected with M-Bus for remote reading for water, gas and electricity. Usually the readings are uni-directional. The readings are consumed from a Backend. We also support LoRaWan. With LoRa we are able to have a bi-directional communication. However, they have special interaction patterns because the device might be turned off to save battery.
```

Comment:
```suggestion
Thomas: the devices are connected with M-Bus for remote reading for water, gas and electricity. Those readings are always uni-directional. The readings are consumed from a Backend. We also support LoRaWan. With LoRa we are able to have a bi-directional communication. However, they have special interaction patterns because the device might be turned off to save battery.
```